zfs.go

```diff
@@ -224,24 +235,60 @@ func (d *Dataset) Mount(overlay bool, options []string) (*Dataset, error) {
 // ReceiveSnapshot receives a ZFS stream from the input io.Reader, creates a
 // new snapshot with the specified name, and streams the input data into the
 // newly-created snapshot.
-func ReceiveSnapshot(input io.Reader, name string) (*Dataset, error) {
+// It includes the option "-F" like boolean value.
+func ReceiveSnapshot(input io.Reader, name string, overwrite bool) (*Dataset, error) {
```

Comment:
Changing API is a no go, with the current scheme of things you're going to have to create a new function probably something like `ReceiveSnapshotOptioned` (bettername++) that takes a `map[string]interface{}` which we can check for `overwrite` existence and do the right thing.